game/scripts/vscripts/abilities/twin/boss_twin_twin_empathy.lua

```diff
@@ -0,0 +1,7 @@
+LinkLuaModifier("modifier_boss_twin_twin_empathy_buff", "abilities/twin/modifier_boss_twin_twin_empathy.lua", LUA_MODIFIER_MOTION_NONE)
```

Comment:
Isn't it called in twin.lua?

ABILITY_empathy = thisEntity:FindAbilityByName("boss_twin_twin_empathy")
twin:AddNewModifier(thisEntity, ABILITY_empathy, "modifier_boss_twin_twin_empathy_buff", {})